Installer/config/FanartHandler.Holidays.xml

```diff
@@ -217,6 +231,13 @@
 		<NameUA>Попільна середа</NameUA>
 		<NameDE>Aschermittwoch</NameDE>
 		<NotCelebratedIn>HE</NotCelebratedIn>
+		<NotCelebratedIn>AU</NotCelebratedIn>
+	</Holiday>
+	<Holiday name="Daylight Savings Time Ends" shortname="DaylightSavings|DaylightSavingsEnd">

```

Comment:
Isn't the current universal implementation of calculation based on computer settings suitable for you? In my opinion, it takes into account all options if the Locale in Windows is configured correctly.

Reply:
Yes, I guess you are right. The problem I have is that in Queensland we don't have daylight saving, but in other states we do so I was hoping to see when the other states started and finished daylight savings.